src/machine/behavior/is/other/get.rs

```diff
@@ -1,6 +1,17 @@
 use machine::state::State;
+use machine::state::sr::R;
 
-pub fn get(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
-}
+pub fn get(state: &mut State, x: u8, y: u8, z: u8) {
+    // Load operands
+    let r = y.wrapping_add(z);
+
+    // Execute
+    if r > 32 {
+    	panic!("no register");
```

Comment:
What's the real behaviour here? Trap or wrap? (Just curious, I'm not expecting a complete implementation of traps and interrupts)